.travis.yml

```diff
@@ -71,7 +71,7 @@ before_script:
   - drupal-ti before_script
 
 script:
-  - phpcs . -p --standard=Drupal --colors --extensions=php,inc,test,module,install
+  - phpcs . -p --standard=Drupal --colors
   - drupal-ti script
```

Comment:
The drupal standard does not by default test module and install files... So instead of moving the --extensions we should add css and js to the list.